cyberark-secrets-provider-for-kubernetes/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+appVersion: "1.0.0"
+description: A Helm chart for Kubernetes
+name: cyberark-secrets-provider-for-kubernetes
```

Comment:
this will also be the name of the container. We can change this to `cyberark-secrets-provider` as we have it for the init container. what do you think? @orenbm @InbalZilberman 
<img width="651" alt="Screen Shot 2020-08-03 at 9 51 38 AM" src="https://user-images.githubusercontent.com/19418506/89154155-062e3880-d56f-11ea-8a1f-cc61497bb5bb.png">

Reply:
i would change `cyberark-secrets-provider` to `cyberark-secrets-provider-for-kubernetes`

Reply:
unless it breaks compatibility in a way that i am missing

Reply:
I can just hard code that in the manifest

Reply:
@InbalZilberman  can we call the chart `cyberark-secrets-provider-for-k8s` to save on line space and considering it will sit in the `secrets-provider-for-k8s` repo

Reply:
@sigalsax yes :)

Reply:
Why is the part `for-kubernetes` needed? Doesn't it go without saying that a Helm chart and a running pod are Kubernetes related? I understand that this is the official name from GitHub, but in GitHub it makes sense because here you don't have the Kubernetes context as you do when it's deployed

Reply:
outside of the conversation about which is better to use, we should keep in mind that most k8s resources have a _really_ low maximum length (32 or 64 chars) so brevity is always preferred

Reply:
@InbalZilberman  ^ please see above thread

Reply:
I would suggest to call it secret-provider-for-k8s, because there is no convention of adding cyberark prefix in other things